modules/effects/arrow-effects-data/src/test/kotlin/arrow/effects/EffectsSuspendDSLTests.kt

```diff
@@ -332,6 +333,24 @@ class EffectsSuspendDSLTests : UnitSpec() {
       } shouldBe false
     }
 
+    "List.parTraverse should run effects in parallel" {
+      fxTest {
+        fx {
+          val timeline = mutableListOf<String>()
+          val tasks = (0 until 3).map {
+            effect {
+              timeline.add("start $it")
+              delay(100)
```

Comment:
I'd multiply the delay by the position (or the inverse), so you know deterministically who's supposed to end first.